### DISCUSSION TITLE
Changes must be made through a pull request.

### DISCUSSION THREAD

**Discussion by yesh4gvm** [community/community#59511] [Pull Requests]
### Select Topic Area

Bug

### Body

Well I am facing some issue i reported on stackoverflow its related to my github account.

I am able to commit and push normal branches but unable to push protected branches with Pull Request which has been already created still it says "Changes must be done through pull request". I am not part of that organization but they have added my user as to bypass the authentication to approve merge code into their private branches. (I have this bypass rule Protect matching branches -> Require a pull request before merging -> Allow Specified actors to bypass required pull requests.) Also we have rule for providing approval on pull request which is completely provided by teammates.

I am able to merge and approve code into protected branches although I am not able to push the protected branch from my local when we do RMC ( Resolve Merge Conflict). Its strange acting this from past few days, I am assuming the new changes are affecting.

Also this repo is private repo so unable to discuss more for now but it surely hits me bad and I need to ask my teammates to resolve the RMC which time kills for both of us.

Let me know if anything else is required from myside.

```
remote: error: GH006: Protected branch update failed for refs/heads/preprod.
remote: error: Changes must be made through a pull request.
```

**Comment by timnolte** [comment]
Was there every any sort of fix for this? We are running into this but with GitHub Actions.

**Reply by yesh4gvm** [reply]
Maybe this will help you - https://stackoverflow.com/q/76512498/17731528
